networks/2_get_network_communities.py

```diff
@@ -36,19 +36,12 @@
 
 # +
 # Params
-corr_threshold = 0.5
+# Choices: {"dbscan", "hierarchal", "affinity"}
+cluster_method = "affinity"
 
 # Correlation matrix files
-pao1_corr_filename = os.path.join(
-    paths.LOCAL_DATA_DIR, f"pao1_corr_{corr_threshold}.tsv"
-)
-pa14_corr_filename = os.path.join(
-    paths.LOCAL_DATA_DIR, f"pa14_corr_{corr_threshold}.tsv"
-)
-
-# +
-## TO DO
-# Create variable to determine which clustering method to use
+pao1_corr_filename = os.path.join(paths.LOCAL_DATA_DIR, "pao1_log_spell_mat.tsv")
+pa14_corr_filename = os.path.join(paths.LOCAL_DATA_DIR, "pa14_log_spell_mat.tsv")
```

Comment:
Minor: you might want to switch to pathlib in the future if you consider it more clear.

Reply:
Nice catch. I'll add these